examples/ci_test.rs

```diff
@@ -8,6 +8,8 @@
 
 //! Run a local network with 'nodes' nodes sending 'requests' requests.
 
+/* TODO: This example makes no sense without client, remove it
```

Comment:
I would vote we do remove this in this PR actually. Saves time and clients (SCL) should do these kind of tests along with vaults themselves.